examples/basic-handler/vite.config.js

```diff
@@ -1,13 +1,18 @@
 import { viteRuntimeNode } from 'vite-runtime-node-plugin';
 import { viteRuntimeWorkerd } from 'vite-runtime-workerd-plugin';
 
-export function basicHandler({ entrypoint }) {
+export function basicHandler({ entrypoint, serverRuntimeId }) {
   return {
     name: 'basic-handler-plugin',
+    /** @param {import('vite').ViteDevServer} server */
     configureServer(server) {
       return async () => {
-        const ssrRuntime = await server.ssrRuntime$;
-        const dispatchRequest = await ssrRuntime.createRequestDispatcher({
+        // Note: here we create an instance of the chosenRuntime, interestingly we can:
+        //       - create more instances of the same runtime (with maybe different options for example)
```

Comment:
I'm realizing only now as I read this, that in order to create a runtime with different options, we'd need refactor the `basicHandler` function to accept an instance of ServerRuntime rather than (or in addition to) the id. That way the app developer can configure the server runtime to their hearts content and pass it in, or just use the runtime name and get a default instance.

let's discuss this a bit more when we talk tomorrow.

Reply:
yes exactly, I made the options optional and in this simple example the runtimes we define just happen not to specify and option, but they could

Reply:
I've updated this to allow and showcase how to for example tweak the `inspectorPort` of Miniflare

Hopefully this is along the lines of what you were thinking? 🙂